From: Markus Trippelsdorf <markus@trippelsdorf.de>
Subject: Fix misleadingly indented assignment (whitespace)
Date: Mon, 14 Dec 2015 16:44:40 +0100
Origin: http://article.gmane.org/gmane.linux.kernel/2108042

Fix misleadingly indented assignment.
This is just a simple whitespace fix.

The issue was pointed out by gcc-6's -Wmisleading-indentation.

Acked-by: Ingo Molnar <mingo@kernel.org>
Signed-off-by: Markus Trippelsdorf <markus@trippelsdorf.de>
---
--- a/tools/perf/util/pmu.c
+++ b/tools/perf/util/pmu.c
@@ -153,7 +153,7 @@ static int perf_pmu__parse_unit(struct p
 	if (fd == -1)
 		return -1;
 
-		sret = read(fd, alias->unit, UNIT_MAX_LEN);
+	sret = read(fd, alias->unit, UNIT_MAX_LEN);
 	if (sret < 0)
 		goto error;
 
